docs/en/04-supported-codebases/mmdet.md

```diff
@@ -1,30 +1,175 @@
-|       YOLOX        |   ObjectDetection    |      Y      |    Y     |  Y   |   N   |    Y     |    [config](https://github.com/open-mmlab/mmdetection/tree/master/configs/yolox)     |
-|   Cascade R-CNN    |   ObjectDetection    |      Y      |    Y     |  N   |   Y   |    Y     | [config](https://github.com/open-mmlab/mmdetection/tree/master/configs/cascade_rcnn) |
-|    Faster R-CNN    |   ObjectDetection    |      Y      |    Y     |  Y   |   Y   |    Y     | [config](https://github.com/open-mmlab/mmdetection/tree/master/configs/faster_rcnn)  |
-| Faster R-CNN + DCN |   ObjectDetection    |      Y      |    Y     |  Y   |   Y   |    Y     | [config](https://github.com/open-mmlab/mmdetection/tree/master/configs/faster_rcnn)  |
-|        GFL         |   ObjectDetection    |      Y      |    Y     |  N   |   ?   |    Y     |     [config](https://github.com/open-mmlab/mmdetection/tree/master/configs/gfl)      |
-|     RepPoints      |   ObjectDetection    |      N      |    Y     |  N   |   ?   |    Y     |  [config](https://github.com/open-mmlab/mmdetection/tree/master/configs/reppoints)   |
-|        DETR        |   ObjectDetection    |      Y      |    Y     |  N   |   ?   |    Y     |     [config](https://github.com/open-mmlab/mmdetection/tree/master/configs/detr)     |
-| Cascade Mask R-CNN | InstanceSegmentation |      Y      |    N     |  N   |   N   |    Y     | [config](https://github.com/open-mmlab/mmdetection/tree/master/configs/cascade_rcnn) |
-|     Mask R-CNN     | InstanceSegmentation |      Y      |    Y     |  N   |   N   |    Y     |  [config](https://github.com/open-mmlab/mmdetection/tree/master/configs/mask_rcnn)   |
-|  Swin Transformer  | InstanceSegmentation |      Y      |    Y     |  N   |   N   |    N     |     [config](https://github.com/open-mmlab/mmdetection/tree/master/configs/swin)     |
+# MMDetection Deployment
+
+[MMDetection](https://github.com/open-mmlab/mmdetection) aka `mmdet` is an open source object detection toolbox based on PyTorch. It is a part of the [OpenMMLab](https://openmmlab.com/) project.
+
+## Installation
+
+### Install mmdet
+
+If you have already done that, move on to [the next section.](#install-mmdeploy) Otherwise, please follow the [guide](https://mmdetection.readthedocs.io/en/3.x/get_started.html) to finish mmdet installation.
+
+### Install mmdeploy
+
+There are several methods to install mmdeploy, among which you can choose an appropriate one according to your target platform and device.
+
+**Method I:** Install precompiled package
+
+> **TODO**. MMDeploy hasn't released based on dev-1.x branch.
+
+**Method II:** Build using scripts
+
+If your target platform is **Ubuntu 18.04 or later version**, we encourage you to run
+[scripts](../01-how-to-build/build_from_script.md). For example, the following commands help install mmdeploy as well as inference engine - `ONNX Runtime` automatically.
```

Comment:
For example, the following commands help install mmdeploy --》For example, the following commands install mmdeploy and the inference engine -`ONNX Runtime`.